deepquest/__main__.py

```diff
@@ -33,6 +40,19 @@ def score(args):
     from . import score
     score(args.files)
 
+def set_gpu_id(gpuid):
+    import os
+    os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
+    gpuid = gpuid.split(',') if ',' in gpuid else gpuid.split()
+    gpustr = ''
+    gpucount = 0
+    for g in gpuid:
+        gpustr += str(g).strip() + ','
+        gpucount += 1
+    gpustr = gpustr[0:-1]
+    os.environ["CUDA_VISIBLE_DEVICES"] = gpustr
+
+    return gpucount
```

Comment:
Similary to `change2dict()` that was moved to `utils/utils.py`, it is probably cleaner to move this function under `utils` as well, w.r.t. to the main functions `train()`, `predict()` and `score()`.

Reply:
Moved to `__init__.py` rather than `utils/utils.py` as `set_gpu_id()` has to happen before `import deepquest`. Commit 7cb09bd